CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+0.4.7 (June 2019)
+-----------------
+- update to OMERO 5.5
+- preserve backward compatibility to OMERO 5.4
+
 0.4.6 (June 2019)
 -----------------
 - fixed bug in Image.getPixelValues() method which returned
```

Comment:
maybe remove usage of past tense for previous release

Reply:
I'll use past tense, too; looks like it's always been past tense for the previous releases.